src/Appwrite/Utopia/Response/Model/Identity.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class Identity extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('$id', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Identity ID.',
+                'default' => '',
+                'example' => '5e5ea5c16897e',
+            ])
+            ->addRule('$createdAt', [
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Identity creation date in ISO 8601 format.',
+                'default' => '',
+                'example' => self::TYPE_DATETIME_EXAMPLE,
+            ])
+            ->addRule('$updatedAt', [
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Identity update date in ISO 8601 format.',
+                'default' => '',
+                'example' => self::TYPE_DATETIME_EXAMPLE,
+            ])
+            ->addRule('userId', [
+                'type' => self::TYPE_STRING,
+                'description' => 'User ID.',
+                'default' => '',
+                'example' => '5e5bb8c16897e',
+            ])
+            ->addRule('provider', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Identity Provider.',
+                'default' => '',
+                'example' => 'email',
+            ])
+            ->addRule('status', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Connection status. Can be connected or disconnected',
+                'default' => '',
+                'example' => 'connected',
+            ])
+            ->addRule('providerUid', [
+                'type' => self::TYPE_STRING,
+                'description' => 'ID of the User in the Identity Provider.',
+                'default' => '',
+                'example' => '5e5bb8c16897e',
+            ])
+            ->addRule('providerEmail', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Email of the User in the Identity Provider.',
```

Comment:
A bit skeptic about having email here. OAuth1 might not provide email.

If we use email, what do we do about OAUth1 providers?
Id we dotn use email, do we really need to store it?